.github/ISSUE_TEMPLATE/content-bug.yml

```
@@ -0,0 +1,51 @@
+name: "Content issue"
+description: Report a problem with content on MDN.
+labels: ["needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Before you start
+        **Want to change a page yourself?** This content is open source!
+        ↩ Look for the _Edit on GitHub_ link on any MDN page.
+        **Is your issue about a browser compatibility table?**
+        ↩ Use the _Report problems with this compatibility data on GitHub_ link next to a compatibility table.
+        **Need help with a browser?**
+        🙋 To get help with [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
```

An suggestion: to show localized page, we should remove the language param in URL query:

```suggestion
        🙋 To get help with [Firefox](https://support.mozilla.org/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
```

But I'm not really sure about the link to Firefox, it just works for me (zh-CN).